### DISCUSSION TITLE
2007/01/08/%E3%80%8A%E6%BB%A1%E5%9F%8E%E5%B0%BD%E5%B8%A6%E9%BB%84%E9%87%91%E7%94%B2%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#137] [Announcements]
# 2007/01/08/%E3%80%8A%E6%BB%A1%E5%9F%8E%E5%B0%BD%E5%B8%A6%E9%BB%84%E9%87%91%E7%94%B2%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

本来我这里登陆 live space 就慢，前一阵台湾地震搞得这个网站基本上不来了。所以好一阵都没更新 blog。

https://blog.qizhen.xyz/2007/01/08/%E3%80%8A%E6%BB%A1%E5%9F%8E%E5%B0%BD%E5%B8%A6%E9%BB%84%E9%87%91%E7%94%B2%E3%80%8B%E8%A7%82%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：


wuximing
2013年05月7日 00:45 编辑
先学习，再创造。这个道理再不同领域都适用。

回复
Moca
2007年03月1日 12:13 编辑
最讨厌看翻拍的东西
就像吃昨晚的剩饭

回复
Unknown
2007年01月11日 10:56 编辑
的确不好看，老某应该去排A片
